torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -181,12 +275,26 @@ class ZeroRedundancyOptimizer(Optimizer, _Joinable):
 
     """
 
+    functional_optim_map = {
```

Comment:
This map also lives in `DistributedOptimizer`. Shall we consolidate these two into a single source of truth?

https://github.com/pytorch/pytorch/blob/73f1e2d1dcea810d3655d2a225bb39edf83025a0/torch/distributed/optim/optimizer.py#L196-L209